modules/storage-azure-file-create-storage-class.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent-storage-azure-file.adoc
+
+[id="storage-create-azure-file-storage-class_{context}"]
+= Creating the Azure storage class
+
+StorageClasses are used to differentiate and delineate storage levels and
+usages. By defining a storage class, users can obtain dynamically provisioned
+persistent volumes.
+
+.Additional References
+
+* https://kubernetes.io/docs/concepts/storage/storage-classes/#azure-file[Azure File Storage Class]
+
+.Procedure
+
+. In the {product-title} console, click *Storage* -> *Storage Classes*.
+
+. In the storage class overview, click *Create Storage Class*.
+
+. Define the desired options on the page that appears.
```

Comment:
> :no_entry_sign: desired adj
Do not use. Use “required” or “the <item> that you want.”

> ! appear v
Do not use unless there is no better alternative. Possible alternatives include “is displayed” and “opens.”